downstream/modules/dev-guide/con-content-workflows.adoc

```diff
@@ -0,0 +1,20 @@
+
+[id="con-content-workflows_introduction"]
+
+
+= About content workflows
+
+
+[role="_abstract"]
+Before {PlatformName} 2.0, an automation content developer may have needed so many Python virtual environments that they required their own automation in order to manage them. To reduce this level of complexity, {PlatformNameShort} 2.0 is moving away from virtual environments and using containers, referred to as {ExecEnvName}, instead, as they are straightforward to build and manage and are more shareable across teams and orgs.
```

Comment:
"2.0 is moving away from virtual environments " 
- for module re-use in future AAP versions, maybe change this phrase to something like: "Since version 2.0, {PlatformNameShort} uses containers",